rules.txt

```diff
@@ -855,11 +855,44 @@
 /services/cars-parking-transportation/apply-for-a-temporary-no-parking-permit/ 301  /services/cars-parking-transportation/apply-for-a-parking-permit/apply-for-a-temporary-no-parking-permit/
 /services/cars-parking-transportation/get-your-car-back-when-it-has-been-towed/ 301  /services/cars-parking-transportation/towing/get-your-car-back-when-it-has-been-towed/
 /services/cars-parking-transportation/report-predatory-towing/ 301  /services/cars-parking-transportation/towing/report-predatory-towing/
-/services/crime-law-justice/find-an-inmate/  301  /services/crime-law-justice/inmates/find-an-inmate/
-/services/crime-law-justice/inmates/visit-a-person-in-jail-or-prison/  301 /services/crime-law-justice/inmates/visit-an-inmate/
-/services/crime-law-justice/inmates/send-mail-to-someone-in-jail-or-prison/ 301 /services/crime-law-justice/inmates/contact-an-inmate-by-mail/
-/services/crime-law-justice/inmates/find-an-inmate/ 301 /services/crime-law-justice/inmates/locate-an-inmate/
+/services/crime-law-justice/find-an-inmate/  301  /services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/locate-an-incarcerated-person/
+/services/crime-law-justice/inmates/visit-a-person-in-jail-or-prison/  301  /services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/visit-an-incarcerated-person/
+/services/crime-law-justice/inmates/send-mail-to-someone-in-jail-or-prison/ 301  /services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/contact-an-incarcerated-person-by-mail/
+/services/crime-law-justice/inmates/find-an-inmate/ 301  /  services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/locate-an-incarcerated-person/
```

Comment:
Looks like there's an extra space after the `301 / `

Reply:
On line 861

Reply:
fix'd